scripts/mint-nft.js

```diff
@@ -8,8 +8,8 @@ const PUBLIC_KEY = process.env.PUBLIC_KEY;
 const PRIVATE_KEY = process.env.PRIVATE_KEY;
 
 const config = {
-    apiKey: "N5s7eE3IXH1Udfxs2P2RtRa0G_1SSt-V",
-    network: Network.ETH_RINKEBY,
+    apiKey: "GIsNWI5BbuALsudQ9IYNnHmM-6bW5iBp",
```

Comment:
for now can you make this as a part of config in env file.
Later we can supply these values as part of secret from GitHub